src/ui/screens/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,192 @@
+                            Layout.alignment: Qt.AlignTop
+                            Layout.topMargin: VPNTheme.theme.vSpacingSmall
+                            spacing: VPNTheme.theme.vSpacing
+
+                            VPNGuideList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: guidesList
+                                title: section.title
+                                description: section.description
+                                customGuideFilter: section.filter
+                            }
+
+                            VPNTutorialList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: tutorialsList
+                                customTutorialFilter: section.filter
+                            }
+                        }
+                    }
+                }
+            },
+
+            // Tutorials
+            VPNFlickable {
+                flickContentHeight: layoutTutorial.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutTutorial
+
+                    anchors.fill: parent
+                    anchors.topMargin: VPNTheme.theme.vSpacing
+                    anchors.bottomMargin: VPNTheme.theme.vSpacing
+                    anchors.leftMargin: VPNTheme.theme.windowMargin
+                    anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                    VPNTutorialList { 
+                        Layout.fillWidth: true
+                    }
+                }
+            },
+
+            // Tips
+            VPNFlickable {
+                flickContentHeight: layoutGuide.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutGuide
+
+                    anchors.fill: parent
+                    anchors.topMargin: VPNTheme.theme.vSpacing
+                    anchors.bottomMargin: VPNTheme.theme.vSpacing
+                    anchors.leftMargin: VPNTheme.theme.windowMargin
+                    anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                    Repeater {
+                        model: tipsSections.count
+
+                        Column {
+                            property var section: tipsSections.get(index)
+
+                            visible: !!guidesList.count
+
+                            Layout.fillWidth: true
+                            Layout.alignment: Qt.AlignTop
+
+                            VPNGuideList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: guidesList
```

Comment:
Not sure if it will cause any errors, but I think it's best if we don't reuse id's within the same component